pkg/fanal/test/integration/library_test.go

```diff
@@ -117,6 +117,16 @@ var tests = []testCase{
 			Family: "suse linux enterprise server",
 		},
 	},
+	{
```

Comment:
Since you already added tests in `integration/*_test.go`, you don't have to add tests here.
These tests were created when it used to be a separate library called fanal, but are now part of Trivy and are not well maintained.

Reply:
do you want me to remove it, or should i leave it in

Reply:
Do you mind removing it? I'm sorry to waste your effort.

Reply:
just from pkg/fanal/test/integration/library_test.go or also other places there?

Reply:
A test case from library_test.go and golden files.